app/lnd/config/index.js

```diff
@@ -51,8 +52,9 @@ switch (plat) {
     break
 }
 
-if (process.env.NODE_ENV === 'development') {
-  lndPath = join(appRootPath, 'resources', 'bin', plat, lndBin)
+if (isDev) {
```

Comment:
Can you explain why this change?

Reply:
When you run `npm start` for example, we run the app with `NODE_ENV=production` so in this case that would trigger the second condition which would result in it trying to load the binary from the resources directory which is not what we want.

`isDev` (https://github.com/sindresorhus/electron-is-dev) is better able to detect wether you are running in development or not. NODE_ENV==='development' !== electron running in development.